File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {

        if(nums.length==0) return 0;

        int x = 0;

        for(int i=1;i<nums.length;i++){
            if(nums[i] != nums[x]){
                x = x + 1;
                nums[x] = nums[i];
            }
        }
        return x + 1;
        
     
        
    }
}

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {

    let x = 0;
    let n = nums.length;
    for(let i=0;i<n;i++){
        if(nums[i]>nums[x]){
            x=x+1;
            nums[x]=nums[i];
        }
    }
    return x+1;
    
};